packages/collector/test/nativeModuleRetry/test.js

```diff
@@ -133,4 +154,20 @@ function runCopyPrecompiledForNativeAddonTest(agentControls, opts) {
         ]).then(opts.check)
       ));
   });
+
+  describe('validate precompiled binaries', () => {
+    it('should successfully copy the precompiled binaries', async () => {
```

Comment:
Q: Did you run the extended test on main? 

A good approach is to create a PR to only exend the test and then merge into main. And then open another PR and just replace the library.

Reply:
Created a [PR](https://github.com/instana/nodejs/pull/1276)

Reply:
PR to extending the precompiled binaries merged, and rebased the PR.